certutil: add page (#8664)
tldr-pages/tldr: pages/linux/certutil.md

# certutil

> Manage keys and certificate in both NSS databases and other NSS tokens.

> > More information: <https://manpages.ubuntu.com/manpages/xenial/man1/certutil.1.html>.

Reviewer: This software is made by Mozilla not Ubuntu, we prefer manned as link then.
```suggestion
> More information: <https://manned.org/certutil>.
```

- Create a new certificate database:

`certutil -N -d .`

- List all certificates in a database:

`certutil -L -d .`

- List all private keys in a database:

`certutil -K -d . -f pwdfile.txt`

- Import the signed certificate into the requesters database:

> `certutil -A -n "Server-cert" -t ",," -i server.crt -d .`

Reviewer: We use `path/to/file` as placeholder
```suggestion
`certutil -A -n "Server-cert" -t ",," -i {{path/to/file.crt}} -d .`
```

> - To add subject alternative names, use a comma separated list with the option -8 IE:

Reviewer: I can't understand the clarification about the `-8` argument 
```suggestion
- Add subject alternative names to a given certificate:
```
